test/call_conflict/enc1/enc.c

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include <openenclave/internal/tests.h>
+#include "enc1_t.h"
+
+void enc_ecall1(MyStruct s)
+{
+    OE_TEST(s.x == 5);
```

Comment:
Consider locking down the fact that for `enc_ecall`, the global identifier must be the same and the local identifier can be different.